docs/advanced-topics/openshift.asciidoc

```diff
@@ -9,14 +9,15 @@ endif::[]
 
 This page shows how to run ECK on OpenShift.
 
-* <<{p}-openshift-before-you-begin,Before you begin>>
-* <<{p}-openshift-deploy-the-operator,Deploy the operator>>
-* <<{p}-openshift-deploy-elasticsearch,Deploy an Elasticsearch instance with a route>>
-* <<{p}-openshift-deploy-kibana,Deploy a Kibana instance with a route>>
-* <<{p}-openshift-apm,Deploy an APM Server instance with a route>>
-* <<{p}-openshift-beats,Grant privileged permissions to Beats>>
+* <<{p}-openshift-before-you-begin>>
+* <<{p}-openshift-deploy-the-operator>>
+* <<{p}-openshift-deploy-elasticsearch>>
+* <<{p}-openshift-deploy-kibana>>
+* <<{p}-openshift-apm>>
+* <<{p}-openshift-beats>>
 
-NOTE: Only Elasticsearch and Kibana are compatible with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Security Context Constraint]. To run the APM Server on OpenShift you must allow the Pod to run with the `anyuid` SCC as described in <<{p}-openshift-apm>>.
+WARNING: For versions older than 7.9, APM Server and Enterprise Search are not compatible with `restricted` SCC. Use this <<{p}-openshift-apm,workaround>> to overcome this limitation.
```

Comment:
I think we should let the user know that Enterprise Search 7.8 was compatible with Openshift. This is a specific bug in the 7.9 release. Otherwise they might be surprised to have something working with 7.8 and not with 7.9 ?

```suggestion
WARNING: Some Docker images are known to be incompatible with the `restricted` SCC. This is the case for the *APM Server before 7.9* and for the *7.9 release of Enterprise Search*. A <<{p}-openshift-anyuid-workaround,workaround>> is to run those images with the `anyuid` SCC.
```